### ISSUE TITLE
Project Still Running and Taking Up License After Deleted

### ISSUE THREAD

**Issue by Blaec-CoCalc** [sagemathinc/cocalc#7749]
**Describe the bug**
I deleted a CoCalc project called Linear Algebra and created a new project, MTH355. I applied the same license (applied to Linear Algebra) and mistakenly assumed that deleting the project would stop it (since I am the license manager) and prevent the run limit from being hit. 

I noticed the "remove upgrades button" under "deleted projects" from the project dashboard. I just wonder if we should present that immediately as an option when you are actually choosing to delete the project.

**To Reproduce**
1. Create a project
2. Apply a license
3. Delete the project
4. Create a new project
5. Apply the same license
6. Notice the run limit is hit, and you have to manually go into your deleted projects and shut it off (I actually removed and reattached my license to get it to recognize that the run limit was freed up).

**Expected behavior**
I guess I expected a deleted project to automatically be stopped and no longer take up the run limit of my license.

**Screenshots**
If applicable, add screenshots to help explain your problem.  **If the screenshots include text, make sure to also include that text separately so that we can copy and paste it and not have to type it in again!**

<img width="1512" alt="Screenshot 2024-08-08 at 4 47 56 PM" src="https://github.com/user-attachments/assets/a3af4baf-c42b-436b-902e-09ce0b700b17">

**Additional context**
Using Chrome and Incognito.

**Comment by haraldschilly** [comment]
my two cents: maybe we should set a deleted project to be "blocked" immediately – and unblock upon undelete?

**Comment by williamstein** [comment]
> my two cents: maybe we should set a deleted project to be "blocked" immediately – and unblock upon undelete?

Yes that is also worth doing, but I don't know if it will help with the reported issue.  There's nothing in this codebase that does anything with blocked.  It's only a thing in kucalc (in the manage scripts).
